frontend/src/components/containers/ContainersDetailContent.tsx

```diff
@@ -74,6 +70,9 @@ const ContainersDetailContent = () => {
         extra={
           !isLoaded ? null :
             <Space>
+              <Button onClick={() => navigate(
+                `/lab-work/samples?${(container?.children.length ?? 0) > 0 ? 'container__location__barcode=' : 'container__barcode='}${container?.barcode}`
```

Comment:
It might be nice (optional) to find a way to disable the button for containers that are not candidates for workflow management (freezers, rooms, drawers, freezer racks, ...)